@fedimint/utils needs to be built before `dev`

```
ERROR in src/setup/FederationSetup.tsx:14:32
TS2307: Cannot find module '@fedimint/utils' or its corresponding type declarations.
    12 | import { VerifyGuardians } from '../components/VerifyGuardians';
    13 | import { SetupComplete } from '../components/SetupComplete';
  > 14 | import { useTranslation } from '@fedimint/utils';
       |                                ^^^^^^^^^^^^^^^^^
    15 |
    16 | const PROGRESS_ORDER: SetupProgress[] = [
    17 |   SetupProgress.Start,
```
This happens on a first time run because `@fedimint/utils` hasn't been built yet for the first time. We fixed this same issue with `@fedimint/ui`: https://github.com/fedimint/ui/blob/ad46faa3b7b513b0c8d457e6a2301a1b8bfc9422/turbo.json#L27-L35

--- a/mprocs-nix-gateway.yml
+++ b/mprocs-nix-gateway.yml
@@ -23,8 +23,7 @@ procs:
   devimint:
     shell: tail -n +0 -F $FM_LOGS_DIR/devimint.log
   gateway-ui:
-    cwd: apps/gateway-ui/
-    shell: yarn dev
+    shell: yarn dev:gateway-ui
     env:
       PORT: '3004'
       REACT_APP_FM_GATEWAY_API: 'http://127.0.0.1:8175'

--- a/mprocs-nix-guardian.yml
+++ b/mprocs-nix-guardian.yml
@@ -23,29 +23,25 @@ procs:
   devimint:
     shell: tail -n +0 -F $FM_LOGS_DIR/devimint.log
   guardian-ui-1:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3000'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18138
       BROWSER: none
   guardian-ui-2:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3001'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18140
       BROWSER: none
   guardian-ui-3:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3002'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18142
       BROWSER: none
   guardian-ui-4:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3003'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18144

--- a/mprocs.yml
+++ b/mprocs.yml
@@ -8,22 +8,19 @@ procs:
     shell: docker compose down
     autostart: false
   guardian-ui-1:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3000'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18174
       BROWSER: none
   guardian-ui-2:
-    cwd: apps/guardian-ui/
-    shell: yarn dev
+    shell: yarn dev:guardian-ui
     env:
       PORT: '3001'
       REACT_APP_FM_CONFIG_API: ws://127.0.0.1:18184
       BROWSER: none
   gateway-ui:
-    cwd: apps/gateway-ui/
-    shell: yarn dev
+    shell: yarn dev:gateway-ui
     env:
       PORT: '3002'
       REACT_APP_FM_GATEWAY_API: 'http://127.0.0.1:8175'

--- a/turbo.json
+++ b/turbo.json
@@ -24,14 +24,17 @@
       "cache": false,
       "persistent": true
     },
+    "@fedimint/utils#dev": {
+      "dependsOn": ["^build"]
+    },
     "@fedimint/ui#dev": {
       "dependsOn": ["^build"]
     },
     "@fedimint/guardian-ui#dev": {
-      "dependsOn": ["@fedimint/ui#build"]
+      "dependsOn": ["@fedimint/utils#build", "@fedimint/ui#build"]
     },
     "@fedimint/gateway-ui#dev": {
-      "dependsOn": ["@fedimint/ui#build"]
+      "dependsOn": ["@fedimint/utils#build", "@fedimint/ui#build"]
     },
     "clean": {
       "cache": false

